### PR TITLE
adding selenium tests for onboarding

### PR DESCRIPTION
**Pull request by ehfeng**

### CODE REVIEW

**Review by MaxBittker — CHANGES_REQUESTED** [review]
i'd like this to check the project gets created and or that the second button click brings you somewhere

File: tests/acceptance/test_onboarding.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        self.browser.wait_until_not('.loading-indicator')
+        self.browser.snapshot(name='onboarding-choose-platform')
+
+        self.browser.click('.platform-tile.javascript')
+        self.browser.click('.btn-primary')
+
+        self.browser.wait_until('.onboarding-Configure')
+        self.browser.snapshot(name='onboarding-configure-project')
+
+        assert Project.objects.filter(organization=self.org).exists()
```
Comment (MaxBittker) [inline tests/acceptance/test_onboarding.py]: could you assert on the name and platform here?

File: tests/acceptance/test_onboarding.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            teams=[self.team],
+        )
+        self.login_as(self.user)
+        self.path = '/onboarding/%s/' % self.org.slug
+
+    def test_onboarding(self):
+        self.browser.get(self.path)
+        self.browser.wait_until('.onboarding-container')
+        self.browser.wait_until_not('.loading-indicator')
+        self.browser.snapshot(name='onboarding-choose-platform')
```
Comment (MaxBittker) [inline tests/acceptance/test_onboarding.py]: we already snapshot this page as `organization onboarding`, you should remove that or test or just use this test to verify something else about the dom (we don't want to snapshot the same thing twice in percy or use it ensure logic)
https://github.com/getsentry/sentry/blob/master/tests/acceptance/test_organization_onboarding.py#L22

File: tests/acceptance/test_onboarding.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        self.browser.click('.btn-primary')
+
+        self.browser.wait_until('.onboarding-Configure')
+        self.browser.snapshot(name='onboarding-configure-project')
+
+        assert Project.objects.filter(organization=self.org).exists()
+
+        self.browser.click('.btn-primary')
+        self.browser.wait_until_not('.loading')
+        self.browser.snapshot(name='onboarding-created-project')
```
Comment (MaxBittker) [inline tests/acceptance/test_onboarding.py]: this is already percy snapshotted as "project issues not configured", you should verify dom with code here

File: tests/acceptance/test_onboarding.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        self.browser.get(self.path)
+        self.browser.wait_until('.onboarding-container')
+        self.browser.wait_until_not('.loading-indicator')
+        self.browser.snapshot(name='onboarding-choose-platform')
+
+        self.browser.click('.platform-tile.javascript')
+        self.browser.click('.btn-primary')
+
+        self.browser.wait_until('.onboarding-Configure')
+        self.browser.snapshot(name='onboarding-configure-project')
```
Comment (MaxBittker) [inline tests/acceptance/test_onboarding.py]: this test is broken and just snapshots a loading bar, you could also make sure it's not already snapshotted somewhere

**Review by MaxBittker — CHANGES_REQUESTED** [review]
these are using percy to verify  behavior and the failures will be trickier to catch or understand. also we should not use percy to snapshot the same page in multiple places, it's for visuals and things that are tricky to test with explicit checks

File: tests/acceptance/test_onboarding.py (hunk 3 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: Sure, I'll remove the snapshot then. I must have misunderstood your comment "that the second button click brings you somewhere".

File: tests/acceptance/test_onboarding.py (hunk 1 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: Fixed

File: tests/acceptance/test_onboarding.py (hunk 2 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: It looks like that test was created after my PR? It feels like it's sort of a subset of this test (doesn't do actual project creation), I'd advise we delete that one.

File: tests/acceptance/test_onboarding.py (hunk 4 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: Fixed, added `self.browser.wait_until_not('.loading-indicator')`.

Doesn't look like it's being snapshotted anywhere else.

File: tests/acceptance/test_onboarding.py (hunk 2 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: Fixed.

File: tests/acceptance/test_onboarding.py (hunk 3 above)
Comment (ehfeng) [reply tests/acceptance/test_onboarding.py]: Fixed.

**Review by MaxBittker — APPROVED** [review]
thanks 🏞

**Pull request merged**